FreeRTOS_ARP.c

```diff
@@ -99,6 +113,18 @@ static TickType_t xLastGratuitousARPTime = ( TickType_t ) 0;
     MACAddress_t xARPClashMacAddress;
 #endif /* ipconfigARP_USE_CLASH_DETECTION */
 
+/** @brief This local variable is used to keep track of number of ARP requests sent and
+ * also to limit the requests to arpIP_CLASH_MAX_RETRIES per arpIP_CLASH_RESET_TIMEOUT_MS
+ * period. */
+static UBaseType_t uxARPClashCounter = 0;
+
+/** @brief The time at which the last ARP clash was sent. */
+static TimeOut_t xARPClashTimeOut;
+
+/** @brief Next defensive request must not be sent for arpIP_CLASH_RESET_TIMEOUT_MS
+ * period. */
+static TickType_t xARPClashTimeoutPeriod = pdMS_TO_TICKS( arpIP_CLASH_RESET_TIMEOUT_MS );
```

Comment:
`xARPClashTimeoutPeriod` is missing a `U` **prefix** because `TickType_t` is always an unsigned number.